openstack/networking/v2/extensions/layer3/addressscopes/urls.go

```diff
@@ -0,0 +1,13 @@
+package addressscopes
+
+import "github.com/gophercloud/gophercloud"
+
+const resourcePath = "addressscopes"
```

Comment:
I think this should be `address-scopes`. The API docs have this and my test environment fails when using `addressscopes`.